.github/workflows/test.yml

```diff
@@ -0,0 +1,86 @@
+name: Run tests
+
+on:
+  push:
+    branches: [main]
+
+  pull_request:
+    branches: [main]
+
+jobs:
+  run-npm-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [19.x]
+        python-version: ["3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+
+      - name: Install Node dependencies
+        run: |
+          npm install
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Python dependencies
```

Comment:
The python dependencies should not be needed in the npm job.

Reply:
We run `pre-commit`, but we can run it in the `Python` job.

Reply:
Ah ok, let me think. Would it be an idea to indeed run in it the Python job, but reverse the order?

Reply:
So that the npm job "needs" the pytest job.